src/tests/main.spec.js

```diff
@@ -0,0 +1,37 @@
+import { showMessage, getDay, getAdultUsers, getRandomUsers} from './main';
+import { days, defaultProduct, money, users } from './constants';
+
+import { expect } from 'chai';
+import chai from 'chai';
+
+describe('showMessage()', () => {
+  it('should work alert with correct text', () => {
+    const text = 'Hello!';
+    
+    expect(showMessage(text)).to.equal(showMessage(text));
```

Comment:
what the reason in this test? `showMessage(text) === showMessage(text)` It is a wrong validation